ui/src/components/Drawer/Drawer.tsx

```diff
@@ -0,0 +1,107 @@
+import * as React from 'react';
+import {styled, useTheme} from '@mui/material/styles';
+import Drawer from '@mui/material/Drawer';
+import List from '@mui/material/List';
+import Divider from '@mui/material/Divider';
+import IconButton from '@mui/material/IconButton';
+import ChevronLeftIcon from '@mui/icons-material/ChevronLeft';
+import ChevronRightIcon from '@mui/icons-material/ChevronRight';
+import ListItem from '@mui/material/ListItem';
+import ListItemButton from '@mui/material/ListItemButton';
+import ListItemText from '@mui/material/ListItemText';
+import {Link} from "react-router-dom";
+import {useAccounts} from "../../contexts/AccountsContext";
+import {useMediaQuery} from "@mui/material";
+
+const routes = [
+  {url: '/', name: 'Home'},
+  {url: '/create', name: 'New Multisig'},
+  {url: '/about', name: 'About'},
+]
+
+const DrawerHeader = styled('div')(({theme}) => ({
+  display: 'flex',
+  alignItems: 'center',
+  padding: theme.spacing(0, 1),
```

Comment:
The framework has a predefined helper that will resolve up to 4 arguments to "spacing" method. The configured scaling factor is 8px, so it will be transpiled into:
```css
padding: 0 8px
```
f.e. in case: padding: theme.spacing(1, 2, 0, 3)
```css
padding: 8px 16px 0 24px
```
P.S. I took those codes from the examples, but we can simplify it using rem units.

Reply:
Sounds good, yeah I imagine it was from somewhere else. I think having padding set directly would be easier to read.